it/list_test.py

```diff
@@ -45,7 +45,7 @@ def test_list_opensearch_plugins(cfg):
 def test_list_workloads(cfg):
     assert it.osbenchmark(cfg, "list workloads") == 0
     assert it.osbenchmark(cfg, "list workloads --workload-repository=default "
-                           "--workload-revision=b28c9615d576bcc4beeb6725f4c95ed602b76638") == 0
```

Comment:
This branch in the workloads repo was deleted as part of a separate cleanup effort. This change is not related to the logical changes for this PR